src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/RightCurly/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="RightCurly"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="neverSuppressedChecks" value="RightCurlyCheck" />
```

Comment:
this shouldn't be a never suppressed check. Its violations should be checked by parent/ancestor node which includes `else` or `catch`. Fix from https://github.com/checkstyle/patch-filters/pull/167 should help

Reply:
@baratali yes, Iet's fix #167 first.

Reply:
Fix is delayed https://github.com/checkstyle/patch-filters/pull/167#issuecomment-673761087. We will use neverSuppressedChecks for now.